Lib/fontTools/ttLib/ttFont.py

```diff
@@ -1313,9 +1313,7 @@ def getGlyphSet(
             glyphSet = _TTGlyphSetVARC(self, location, glyphSet)
         return glyphSet
 
-    def normalizeLocation(
-        self, location: Mapping[str, _NumberT]
-    ) -> dict[str, _NumberT]:
+    def normalizeLocation(self, location: Mapping[str, _NumberT]) -> dict[str, float]:
```

Comment:
the `_NumberT` trick shouldn't be necessary here because the value type parameter of `Mapping` is covariant